css/properties/-moz-user-focus.json

```diff
@@ -0,0 +1,57 @@
+{
+  "css": {
+    "properties": {
+      "-moz-user-focus": {
```

Comment:
This file is not needed, as it’s covered by `css.properties.user-focus`.

Reply:
I'm unable to find a file for `user-focus` in the repository, or any PRs concerning it.  Additionally, there's only an MDN document for `-moz-user-focus`.  Are you certain?

Reply:
I mistook it for `-moz-user-modify`.

Reply:
+1